docs/typescript/statics-and-methods.md

```diff
@@ -80,3 +80,186 @@ const doc = new User({ name: 'test' });
 // Compiles correctly
 doc.updateName('foo');
 ```
+
+# Using `loadClass()` with TypeScript
+
+Mongoose supports applying ES6 classes to a schema using [`schema.loadClass()`](../api/schema.html#Schema.prototype.loadClass()).
+When using TypeScript, there are a few important typing details to understand.
+
+## Basic Usage
+
+`loadClass()` copies static methods, instance methods, and ES getters/setters from the class onto the schema.
+
+```ts
+class MyClass {
+  myMethod() {
+    return 42;
+  }
+
+  static myStatic() {
+    return 42;
+  }
+
+  get myVirtual() {
+    return 42;
+  }
+}
+
+const schema = new Schema({ property1: String });
+schema.loadClass(MyClass);
+```
+
+Mongoose does **not** automatically update TypeScript types for class members.
+To get full type support, you must manually merge types.
+
+```ts
+interface MySchema {
```

Comment:
Prefer to call this `RawDocType` or something similar since this isn't a mongoose Schema